fedimint-server/src/consensus/server.rs

```diff
@@ -744,8 +734,11 @@ impl ConsensusServer {
 }
 
 pub(crate) async fn get_finished_session_count_static(dbtx: &mut DatabaseTransaction<'_>) -> u64 {
-    dbtx.get_value(&SignedSessionOutcomeCountKey)
+    dbtx.find_by_prefix_sorted_descending(&SignedSessionOutcomePrefix)
```

Comment:
make sure to measure it

Reply:
Even if it responds here fast, it might have a higher underlying cost which will be hard to measure here. (see other comment).